backend-api/src/functions/jwks/jwksHandler.ts

```diff
@@ -0,0 +1,89 @@
+import { CloudFormationCustomResourceEvent, Context } from "aws-lambda";
+import { ConfigService } from "./configService/configService";
+import { dependencies, IJwksDependencies } from "./handlerDependencies";
+
+export async function lambdaHandlerConstructor(
+  dependencies: IJwksDependencies,
+  event: CloudFormationCustomResourceEvent,
+  context: Context,
+): Promise<void> {
+  const logger = dependencies.logger();
+  logger.addContext(context);
+  logger.log("STARTED");
+
+  const resultSender = dependencies.customResourceResultSender(event, context);
+
+  if (event.RequestType === "Delete") {
+    const sendEventResponse = await resultSender.sendEvent("SUCCESS");
+    if (sendEventResponse.isError) {
```

Comment:
I'm logging this block about five times in this file, and I don't like that. I would have preferred passing the logger to `CustomResourceResultSender` and doing all the logging in that class, but I know that the team prefers logging in the handler. Maybe this is something to discuss at some point?